projects/experimental/components/badge/badge.component.ts

```diff
@@ -0,0 +1,93 @@
+import {AnimationOptions} from '@angular/animations';
+import {
+    ChangeDetectionStrategy,
+    Component,
+    HostBinding,
+    Inject,
+    Input,
+} from '@angular/core';
+import {TUI_PLATFORM, tuiIsNumber, tuiIsPresent, TuiPlatform} from '@taiga-ui/cdk';
+import {
+    MODE_PROVIDER,
+    TUI_ANIMATION_OPTIONS,
+    TUI_MODE,
+    TuiBrightness,
+    tuiExpressiveEntrance,
+    TuiSizeS,
+    TuiSizeXL,
+} from '@taiga-ui/core';
+import {TuiBadgeAppearance} from '@taiga-ui/experimental/types';
+import {PolymorpheusPrimitive} from '@tinkoff/ng-polymorpheus';
+import {Observable} from 'rxjs';
+
+@Component({
+    selector: 'tui-badge',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    templateUrl: './badge.template.html',
+    styleUrls: ['./badge.style.less'],
+    providers: [MODE_PROVIDER],
+    animations: [tuiExpressiveEntrance],
+    host: {
+        '($.data-mode.attr)': 'mode$',
+    },
+})
+export class TuiBadgeComponent {
+    @Input()
+    value: PolymorpheusPrimitive;
+
+    @Input()
+    @HostBinding('attr.data-size')
+    size: TuiSizeS | TuiSizeXL = 'l';
+
+    @Input()
+    @HostBinding('attr.data-appearance')
+    appearance: TuiBadgeAppearance = 'default';
+
+    @Input()
+    @HostBinding('class._hoverable')
+    hoverable = false;
+
+    @HostBinding('attr.data-platform')
+    platform = this.tuiPlatform;
+
+    @HostBinding('@tuiExpressiveEntrance')
+    readonly entrance = this.animationOption;
```

Comment:
Do we want it all the time? Maybe just for the notification appearance?

Reply:
discussed with [zergunteg](https://github.com/zergunteg), he said that this animation is for all badges (users can disable it)

Reply:
How do they disable it? Also this prop unnecessary, can move the binding to `host`

Reply:
`@.disabled`  ?

Reply:
after discussion we decided to remove default animation